example/README.md

```diff
@@ -0,0 +1,71 @@
+# Using go-zserio
+
+The instructions below assume, that you are starting fresh, which can be done by simply:
+
+```bash
+$ rm -f go.mod go.sum
+```
+
+Then we can create a new go module for your project:
+
+```bash
+$ go mod init myproject.home/zserio-example
+```
+
+You can use `go-zserio` by installing the CLI tool system-wide, or you can add a `tools/tools.go` file
+with the following:
+
+```go
+//go:build tools
+// +build tools
+
+package tools
+
+import (
+	_ "github.com/woven-planet/go-zserio/cmd/zserio"
+)
+```
+
+This will ensure, that `go mod tidy` will pull in the dependencies for
+`go-zserio` code generator. To use id, one just needs to add:
+
+```go
+//go:generate go run github.com/woven-planet/go-zserio/cmd/zserio generate --rootpackage myproject.home/zserio-example --out . ./schema
+```
+
+And create a `<out_dir>/contacts/dummy.go` file in order to ensure that `go mod` works for developers, who have not yet generated the code. Before the code
+is generated, `go mod` does not know about the code generated by the `zserio`
+code generator and the file helps to `go mod` to resolve the unknown import to
+the local code.
```

Comment:
This paragraph could use a bit of line wrapping

Reply:
Funny enough, this is `pre-commit` doing this to me. Reworded a bit and it worked.